vm/src/memory/unified.rs

```diff
@@ -355,6 +355,39 @@ impl UnifiedMemory {
             _ => Err(MemoryError::UndefinedMemoryRegion),
         }
     }
+
```

Comment:
Are these functions being used anywhere anymore? It seems like no. Either they should be used (rather than making the stores themselves `pub` in the underlying struct), or they should be removed.